tests/integration/components/sl-textarea-test.js

```diff
@@ -17,3 +17,57 @@ test( 'for attribute value on label matches id of textarea', function( assert )
         wrapper.find( 'textarea' ).attr( 'id' )
     );
 });
+
+test( '"spellcheck" property is supported', function( assert ) {
+    this.render( hbs`
+        {{#sl-textarea spellcheck=true}}
+        {{/sl-textarea}}
+    ` );
+
+    const wrapper = this.$( '>:first-child' );
+
+    assert.strictEqual(
+        wrapper.find( 'textarea' ).attr( 'spellcheck' ),
+        'true',
+        'Textarea spellcheck attribute is expected value'
+    );
+});
+
+test( '"spellcheck" property is supported with bound values', function( assert ) {
+    this.set( 'spellcheck', true );
+    this.render( hbs`
+        {{#sl-textarea spellcheck=spellcheck}}
+        {{/sl-textarea}}
+    ` );
+
+    const wrapper = this.$( '>:first-child' );
+
+    assert.strictEqual(
+        wrapper.find( 'textarea' ).attr( 'spellcheck' ),
+        'true',
+        'Textarea spellcheck attribute is expected value'
+    );
+
+    this.set( 'spellcheck', false );
+
+    assert.strictEqual(
+        wrapper.find( 'textarea' ).attr( 'spellcheck' ),
+        'false',
+        'Textarea spellcheck attribute is expected value'
+    );
+});
+
+test( '"spellcheck" property defaults correctly', function( assert ) {
+    this.render( hbs`
+        {{#sl-textarea}}
+        {{/sl-textarea}}
+    ` );
+
+    const wrapper = this.$( '>:first-child' );
+
+    assert.strictEqual(
+        wrapper.find( 'textarea' ).attr( 'spellcheck' ),
```

Comment:
Replace `wrapper` with `this.$( '>:first-child' )` to be consistent with the rest of the codebase.